pubspec.yaml

```diff
@@ -18,6 +18,14 @@ dependencies:
     path: packages/data
   cupertino_icons: ^1.0.2
   firebase_crashlytics: ^2.8.13
+  window_size:
+    git:
+      url: https://github.com/google/flutter-desktop-embedding.git
+      path: plugins/window_size
+      ref: a738913c8ce2c9f47515382d40827e794a334274
```

Comment:
looks interesting

Reply:
the first and more popular solution when u looking for how to set window size: plugin created by Google (https://github.com/google/flutter-desktop-embedding/tree/master/plugins/window_size).

Reply:
but its not published